### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by dandelionadia**
What was done: 
* adds footer
* fixes logo padding in the header on the sm

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+
+export const Footer = () => {
+  return (
+    <div>
+      <div className="w-full h-px border-gray-700 border-t inset-y-0 mb-5" />
+      <div className="text-violet-light text-opacity-60 text-xs mb-5">
+        © 2022 Nadiia Ridko. All rights reserved.
```
Comment (kettanaito) [inline src/components/Footer.jsx]: You can also use `new Date().getFullYear()` to automatically display the current year in the copyright notice ;) 
```suggestion
        © {new Date().getFullYear()} Nadiia Ridko. All rights reserved.
```

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      </div>
+      <div className="text-violet-light text-opacity-60 text-xs mb-5">
+        Made by Nadiia and Artem (You're always will have a special place in my
+        hart ❤️).
```
Comment (kettanaito) [inline src/components/Footer.jsx]:
```suggestion
        heart ❤️).
```
❤️

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+export const Footer = () => {
+  return (
+    <div>
+      <div className="w-full h-px border-gray-700 border-t inset-y-0 mb-5" />
+      <div className="text-violet-light text-opacity-60 text-xs mb-5">
+        © 2022 Nadiia Ridko. All rights reserved.
+      </div>
+      <div className="text-violet-light text-opacity-60 text-xs mb-5">
+        Made by Nadiia and Artem (You're always will have a special place in my
```
Comment (kettanaito) [inline src/components/Footer.jsx]:
```suggestion
        Made by Nadiia and Artem (You will always have a special place in my
```

Comment (kettanaito) [reply src/components/Footer.jsx]: If you want, you can mention that it was designed by me.

**Review by kettanaito — CHANGES_REQUESTED** [review]
Just a few comments!

File: src/components/Footer.jsx (hunk 2 above)
Comment (dandelionadia) [reply src/components/Footer.jsx]: 😅

File: src/components/Footer.jsx (hunk 3 above)
Comment (dandelionadia) [reply src/components/Footer.jsx]: 😅

**Review by kettanaito — APPROVED** [review]
😗

**Pull request merged**